src/Psr/CacheItemPool/CacheItemPoolDecorator.php

```diff
@@ -212,40 +220,7 @@ public function save(CacheItemInterface $item)
             throw new InvalidArgumentException('$item must be an instance of ' . CacheItem::class);
```

Comment:
Shouldn't be part of `saveMultipleItems`?

Reply:
Nah, its checked in all methods so we can avoid "surprises" later on.
The typehint within `saveMultipleItems` already ensures that we only pass `CacheItem` to it.

The thing is, that you can also `saveDeferred` items which are not directly triggering the save process. Having the check only in `saveMultipleItems` would lead to deferred exceptions.